tools/cli/main.js

```diff
@@ -616,7 +616,7 @@ Fiber(function () {
   // tight timetable for 1.0 and there is no advantage to doing it now
   // rather than later. #ImprovingCrossVersionOptionParsing
 
-  var isBoolean = { "--help": true, "--unsafe-perm": true };
```

Comment:
I'm going to restore the `--unsafe-perm` option, but have it just set `process.env.METEOR_UNSAFE_PERM = "true"`.